src/OrbitBase/Profiling.cpp

```diff
@@ -38,4 +42,17 @@ uint64_t EstimateClockResolution() {
   }
   return minimum_resolution_found;
 }
+
+uint64_t EstimateAndLogClockResolution() {
+  // We expect the value to be small, ~35 nanoseconds.
```

Comment:
This comment refers to an instance, so it's now out of place now that the function was moved.

Reply:
Modified the comment to also reflect the typical value we get on Windows.